packages/components/src/combobox-control/styles.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Leaving both sass and emotion styles for these components feels a bit weird — although I understand if we want to keep the scope of this PR focused on introducing the new default size.